ReadMe.md

```diff
@@ -127,7 +134,7 @@ English | Japanese | Kana | Notes
 --------|----------|------|------
 Server | サーバ / サーバー | |
 Physical server | 物理サーバ | ぶつりサーバ |
-Virtual server | 仮想サーバ | かそうサーバ |
+Virtual server | 仮想マシン・仮想サーバ | かそうマシン・かそうサーバ | Machine tends to be favoured in most openly available technical documentations (ie. VMWare, etc)
```

Comment:
After checking with @endow, the most natural use of language would warrant 2 entries:

Virtual server: 仮想サーバ
Virtual machine: 仮想マシン

There's no real need to mix the two into a single definition with a ton of nuance, since the natural usage of the Japanese seems to fit pretty much with the natural usage of the English counterparts.